test/unit/pre_checks_test.py

```diff
@@ -4,19 +4,23 @@
     patch, Mock
 )
 
-from suse_migration_services.prechecks.pre_checks import main
+# from suse_migration_services.prechecks.pre_checks import main
```

Comment:
Guessing this was commented out for debug purposes?

Reply:
thanks, will remove